Userland/Libraries/LibWeb/Fetch/BodyInit.cpp

```diff
@@ -43,14 +45,12 @@ WebIDL::ExceptionOr<Infrastructure::BodyWithType> extract_body(JS::Realm& realm,
     }
     // 3. Otherwise, if object is a Blob object, set stream to the result of running object’s get stream.
     else if (auto const* blob_handle = object.get_pointer<JS::Handle<FileAPI::Blob>>()) {
-        // FIXME: "set stream to the result of running object’s get stream"
-        (void)blob_handle;
-        stream = MUST_OR_THROW_OOM(realm.heap().allocate<Streams::ReadableStream>(realm, realm));
+        stream = TRY(blob_handle->cell()->stream());
```

Comment:
Hmm, maybe I should have made this public: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/FileAPI/Blob.h#L59

It's literally the same thing as calling that, but does closer match spec text. Looks good to me though!

By the way, another potential problem we may run into (later on) is this: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/Streams/ReadableStreamDefaultReader.cpp#L87

Reply:
Ah, missed that - went ahead and made that public.